stan/math/opencl/rev/beta.hpp

```diff
@@ -61,6 +61,84 @@ inline auto beta(const T_a& a, const T_b& b) {
   return res;
 }
 
+/**
+ * Return the elementwise `beta()` on an input kernel
+ * generator expression and scalar.
+ *
+ * @tparam T_a type of first expression
+ * @tparam T_b type of scalar
+ * @param a kernel generator expression
+ * @param b scalar
+ * @return elementwise `beta()`
+ */
+template <typename T_a, typename T_b,
+          require_nonscalar_prim_or_rev_kernel_expression_t<T_a>* = nullptr,
+          require_stan_scalar_t<T_b>* = nullptr,
+          require_any_var_t<T_a, T_b>* = nullptr>
+inline auto beta(const T_a& a, const T_b& b) {
+  const arena_t<T_a>& a_arena = a;
+  const arena_t<T_b>& b_arena = b;
```

Comment:
```suggestion
inline auto beta(T_a&& a, const T_b& b) {
  const arena_t<T_a>& a_arena = std::forward<T_a>(a);
```
Scalars don't need arena variables. Matrices should use perfect forwarding. Same for the other overload and other functions.

Reply:
I think all of them should be using perfect forwarding correctly. Ready for another look.